src/System.Threading.AccessControl/tests/AclTests.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace System.Threading.Tests
```

Comment:
Missing the license header:
```suggestion
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information.

namespace System.Threading.Tests
```